recipes/basemap-data-hires/bld.bat

```diff
@@ -0,0 +1,2 @@
+copy %SRC_DIR%\lib\mpl_toolkits\basemap\data\*_h.dat %LIBRARY_PREFIX%\share\basemap || exit 1
+copy %SRC_DIR%\lib\mpl_toolkits\basemap\data\*_f.dat %LIBRARY_PREFIX%\share\basemap || exit 1
```

Comment:
The smaller the better, so :+1: if it is easy to do (only if you are patching basemap anyway)